backend/lib/spree/backend_configuration.rb

```diff
@@ -6,6 +6,20 @@ module Spree
   class BackendConfiguration < Preferences::Configuration
     preference :locale, :string, default: I18n.default_locale
 
+    # @!attribute [rw] themes
+    #   @return [Hash] A hash containing the themes that are available for the admin panel
+    preference :themes, :hash, default: {
+      classic: 'spree/backend/all',
+    }
+
+    # @!attribute [rw] theme
+    #   @return [String] Default admin theme name
+    versioned_preference :theme, :string, initial_value: 'classic', boundaries: { "4.1.0.a" => "classic" }
```

Comment:
Are we going to change this value when we'll introduce the new theme, right?

Reply:
yes!